options.rb

```diff
@@ -0,0 +1,23 @@
+# we will use this file to store our options
+require_relative 'app'
+class Options
+  def option(input)
+    meth = App.new
+    case input
+    when '1'
+      meth.list_all_books
+    when '2'
```

Comment:
- After adding books to the library, when I try to list all books, I still get `Please add a book to the library`. Kindly fix this 👍🏽
![Screenshot from 2023-03-13 16-57-53](https://user-images.githubusercontent.com/44624138/224762359-bf498264-729f-4c6b-adaf-56c0a4c55154.png)